scripts/schemas/twister/testsuite-schema.yaml

```diff
@@ -130,6 +130,11 @@ schema;scenario-schema:
             "regex":
               type: str
               required: true
+            "as_json":
```

Comment:
Not enough of a twister expert to really review here, but it would be good to add a section to the docs somewhere describing this and giving a simple example of usage.

Reply:
right, documentation added.